app/views/layouts/accordion.html.erb

```diff
@@ -17,14 +16,31 @@
                     <%= tag.div(tag.p(@front_matter["alert"]), class: "content-alert content-alert--fullwidth") %>
                   <% end %>
 
-                  <%= render Content::AccordionComponent.new do |accordion| %>
-                    <% @front_matter["steps"]&.each do |title, contents| %>
-                      <%= accordion.slot(:step, title: title, call_to_action: contents["cta"]) do %>
-                        <%= render(partial: contents["partial"]) %>
+                  <%# FIXME: we're tapping the frontmatter here because
```

Comment:
We've got two different work arounds we're using for this in the source code now, I don't think we should block this going out because of it but having FIXME's going in doesn't sit well -- shall we have a think about a longer-term solution we can live with to follow up this PR?

Reply:
I switched the `@parent_front_matter` approach to this one so it's just in one place. Agree we could do with a more robust fix though

Reply:
I changed this back to how it was before